### PR TITLE
Ux review adaptions

### PR DESCRIPTION
**Pull request by mrsimpson**
After reviewing the design with the content team, I adapted some small things which shall make it more easy to read while keeping the comparatively flat design.

WDYT?
diary-ux review

### CODE REVIEW

File: src/client/styles/objects/columns.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.columns {
+    padding: 0px;
```
Comment (jschirrmacher) [inline src/client/styles/objects/columns.scss]: With the value '0' you don't need 'px'

File: src/client/styles/objects/columns.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+    margin-right: 0px !important;  
+    margin-left: 0px !important;
```
Comment (jschirrmacher) [inline src/client/styles/objects/columns.scss]: Is `!important` really necessary?

**Review by jschirrmacher — APPROVED** [review]
lgtm, just provided some comments which doesn't prevent merging.

File: src/client/styles/objects/columns.scss (hunk 2 above)
Comment (mrsimpson) [reply src/client/styles/objects/columns.scss:5]: yes, else other scss-variables override the padding

File: src/client/styles/objects/columns.scss (hunk 1 above)
Comment (mrsimpson) [reply src/client/styles/objects/columns.scss:2]: there are some stylint-checks always ensuring a unit. Thus, I always add it

**Pull request merged**